### PR TITLE
Fix profile completion selector for profile pic/cover photo

### PR DESCRIPTION
**Pull request by rickyrombo**
### Description

Fixes a bug where profile pic/cover photo show as completed when unset

### Dragons

Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?

### How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.

Locally against stage

### How will this change be monitored?

For features that are critical or could fail silently please describe the monitoring/alerting being added.

### CODE REVIEW

File: src/common/models/User.ts [hunk 1]
```diff
@@ -60,6 +60,8 @@ export type ComputedUserProperties = {
   _profile_picture_color?: Color
   _artist_pick?: ID
   _has_reposted?: boolean
+  updatedProfilePicture: any
```
Comment (rickyrombo) [inline src/common/models/User.ts:63]: lazily set these to `any` for now...

Comment (raymondjacobson) [inline src/common/models/User.ts:63]: Sorry for review delay. This can be
`{ file: File, url: string } | null`

**Review by raymondjacobson — APPROVED** [review]

File: src/common/models/User.ts (hunk 1 above)
Comment (raymondjacobson) [reply src/common/models/User.ts:63]: or rather `updateProfilePicture?: { file: File, url: string }`

Comment (raymondjacobson) [reply src/common/models/User.ts:63]: the `?` is necessary for the ts compile to work (see CI)

Comment (rickyrombo) [reply src/common/models/User.ts:63]: indeed, i rushed the typing there

**Review by raymondjacobson — APPROVED** [review]

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/mjp-profile-pic-optimism

**Pull request merged**